nx-X11/programs/Xserver/hw/nxagent/Rootless.c

```diff
@@ -497,6 +504,7 @@ int nxagentExportProperty(pWin, property, type, format, mode, nUnits, value)
   #ifdef _XSERVER64
   else if (strcmp(typeS, "CARDINAL") == 0 || strcmp(typeS, "WM_SIZE_HINTS") == 0)
   {
+    /* FIXME: is it okay here to ignore malloc fails? */
```

Comment:
Well given the code is only doing something when `malloc` succeeded... probably okay... If `export` isn't true, the function won't do much later on.